tests/test_about.py

```diff
@@ -45,6 +45,7 @@ def test_about():
     assert "default.gaussian" in out
 
 
+@pytest.mark.skip(reason="This test causes tape mode to be reset")
```

Comment:
Is this true for qchem or just for how this test is coded up? Will we have to rewrite this test soon?

Reply:
It seems to be the use of `importlib.reload(qml)`. This test will pass fine, but all tests after it will be forced into non-tape mode, and I cannot reactivate the tape 🤔

Reply:
Once the old core is deleted, we can remove this decorator and it should be fine.